### ISSUE TITLE
Add the ability for console/stdout logs to automatically include class and method names along with the message

### ISSUE THREAD

**Issue by arbokrad** [jongpie/NebulaLogger#190]
To assist with tracing and debugging, we very often have code that looks something like the following:

```java
public class SomeClass {
  private static final String CLASS_NAME = SomeClass.class.getName();

  private static final String LOG_FORMAT = '{0} -- {1} -- {2}';

  public someMethod() {
    final String METHOD_NAME = 'someMethod()';

    Logger.debug( new LogMessage( LOG_FORMAT, CLASS_NAME, METHOD_NAME, 'Im here!' );

    // ...
  }
}
```

The existing LogMessage makes this very easy to set up, but having hardcoded method names (and less so class names) represents a source of technical debt and verbosity that complicate and distract from the actual non-logging code.

Making the class name and method name available to the logger when it spits out to the debug logs would be great! Perhaps as standardized message parameters like `{className}` and '{methodName}`

**Comment by jongpie** [comment]
Thanks again for logging this! I'm hoping to look into this further in a few weeks, after I've wrapped up a few other in-progress items. I'll let you know once I've made some progress.

**Comment by jongpie** [comment]
Awesome, thank you!! I've promoted the package version, and published [release v4.6.6](https://github.com/jongpie/NebulaLogger/releases/tag/v4.6.6)
